xdsl/transforms/experimental/ConvertStencilToLLMLIR.py

```diff
@@ -9,30 +9,27 @@
 from xdsl.dialects.builtin import ArrayAttr, FunctionType, IntegerAttr, ModuleOp, i64
 from xdsl.dialects.func import FuncOp
 from xdsl.dialects.memref import MemRefType
-from xdsl.dialects import memref
+from xdsl.dialects import memref, arith, scf, builtin
 
-from xdsl.dialects.experimental.stencil import CastOp, FieldType, IndexAttr, LoadOp
+from xdsl.dialects.experimental.stencil import ApplyOp, CastOp, FieldType, IndexAttr, LoadOp, TempType
 
 _TypeElement = TypeVar("_TypeElement", bound=Attribute)
 
 
 def GetMemRefFromField(
-        inputFieldType: FieldType[_TypeElement]) -> MemRefType[_TypeElement]:
-    dims = [i.value.data for i in inputFieldType.shape.data]
+    inputType: FieldType[_TypeElement] | TempType[_TypeElement]
```

Comment:
I think most fields should also be `snake_case`. Although we are quite inconsistent with that. Although most cases in our code where we do this are (hopefully) for external api compatibility (e.g. MLIR might not use snake case for attribute names)

Reply:
Oh, well this is similar here, they're from open earth.
I can switch to snake. Is there a way to check that automatically?

Reply:
Nah, got confused, just corrected!